source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,347 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to specify **sort criteria** for your
+queries using **builders**.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some
+examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest time of day to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance. The builder
```

Comment:
I don't think you use the builder pattern to interact with MongoDB. For all the builders we're documenting in this section, they are used to construct Bson objects.

Reply:
Thanks for the feedback. You are right, it is incorrect to say that you use the builders to interact with a MongoDB instance, they are used to construct Bson objects. Thanks for catching that.

After trying out a few different configurations of this paragraph, I think placing the "builder pattern" definition sentence before the "builders" sentence is the most natural sequence to present these ideas (builder pattern -> builders  are classes to make bson objects -> builders improve code readability). Let me know if you disagree.

Fix: https://github.com/mongodb/docs-java/pull/59/commits/a3b9316117f01736b27daf50bbe7a02f478674c9